cli/command/formatter/history.go

```diff
@@ -79,7 +79,7 @@ func (c *historyContext) ID() string {
 }
 
 func (c *historyContext) CreatedAt() string {
-	return time.Unix(c.h.Created, 0).Format(time.RFC3339)
+	return time.Unix(c.h.Created, 0).UTC().Format(time.RFC3339)
```

Comment:
I'm not sure about this change. What happens without it?

Reply:
This turns out to be not OpenBSD specific now that I dug into it a bit more.

The test works fine if using Docker containers since it seems they're using UTC as the timezone.

However, if run on your local system with a timezone other than UTC, you get a string formatted that doesn't match the expected pattern since `time.Format` uses the local timezone. (So on my machine, it's `2009-11-10T18:00:00-05:00`.)

Reply:
Ok, so I think the formatter is probably correct, but we could adjust the tests to expect the time with the local timezone set, instead of assuming UTC.